Makefile

```
@@ -8,7 +8,7 @@ IMAGE_BUILD_CMD ?= docker build
 IMAGE_BUILD_EXTRA_OPTS ?=
 IMAGE_PUSH_CMD ?= docker push
 CONTAINER_RUN_CMD ?= docker run
-BUILDER_IMAGE ?= golang:1.20-bullseye
+BUILDER_IMAGE ?= golang:1.21-bullseye
```

not important but could move to Debian 12 at the same time?
```suggestion
BUILDER_IMAGE ?= golang:1.21-bookworm
```

---

I am ok with that, @marquiz ?

---

Yeah, let's do this. Follow the tip of the world on mainline :)